src/controllers/cart.controller.ts

```
@@ -0,0 +1,488 @@
+            return;
+        }
+        res.status(500).json({
+            status: 'error',
+            message: 'Failed to add item to cart'
+        });
+    }
+};
+
+// Update cart item
+export const updateCartItem = async (
+    req: Request<{ id: string }, {}, ICartItemUpdate>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)